example-overrides/local-overrides-rhel8_2-tripleo-osp16-1.yaml

```diff
@@ -0,0 +1,15 @@
+standalone_host: <standalone FQDN>
```

Comment:
You're missing `rhos_release: 16.1` somewhere.

Reply:
this is only needed if you want to install from the puddle. Users who want to use the puddle (and not RHSM) would not need all these bits, just to set `rhos_release: 16.1` without RHSM bits.